org.eclipse.jgit.test/tst/org/eclipse/jgit/api/DiffCommandTest.java

```diff
@@ -241,6 +242,31 @@ public void testNoOutputStreamSet() throws Exception {
 		assertEquals("test.txt", diff.getNewPath());
 	}
 
+	@Test
+	/**
+	 * Setting a delta filter of type ANY (applies to any file) with regex that matches the
```

Comment:
can you update the description? it is from the time in which we had filter types. ty

Reply:
updated.